secure-biometric-device/.babelrc.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Lets rename the project and folder to "secure-biometric-interface-integrator"

Reply:
hi @vishwa-vyom , this has been done, it can be checked in [b321370](https://github.com/mosip/esignet-plugins/pull/5/commits/b321370a71e6d926522bb0f944139bf13c4b04c2) commit